dxr/static/js/dxr.js

```diff
@@ -270,7 +270,7 @@ $(function() {
     function countLines(results) {
         var total = 0;
         for (var k = 0; k < results.length; k++) {
-            total += results[k].lines.length;
+            total += Math.max(results[k].lines.length, 1);
```

Comment:
At the least, we should rename countLines, because that's not what it does anymore. But this certainly does fix a bug; good job!